frontend/src/features/help/components/HelpModal.tsx

```diff
@@ -17,53 +19,33 @@ interface ModalProps {
 }
 
 /**
- * This wrapper component allows this dialog to be dragged by clicking and dragging the modal header.
+ * Help component used to display a modal interface with help text and the ability to create a mailto ticket.
+ *
+ * Chrome has added some protection such that mailto links generated by JS cannot be fired without direct user input.
+ * In addition, Chrome caches mailto link values so updating the href after initial load will not be ignored unless the component
+ * with the href is removed from the dom completely.
+ * Displaying the temporary disabled submit button until the mailto address is determined will resolve this issue.
+ *
+ * @param props customize the component with custom text, and an operation to take when the component is closed.
  */
-const DraggableModalDialog = (props: any) => {
-  return (
-    <Draggable handle=".modal-title">
-      <ModalDialog {...props} />
-    </Draggable>
-  );
-};
-
-const DraggableTitle = styled(Modal.Title)`
-  width: 100%;
-  cursor: pointer;
-  color: white;
-`;
+const HelpModal: FunctionComponent<IModalProps> = ({ handleCancel, handleSubmit, show }) => {
```

Comment:
This component is being re-rendered twice for some reason.  Which results in two requests to the API for the tenant settings.  I couldn't debug why and it's out of scope for the work I did.